WooCommerce/Classes/ViewRelated/Products/Media/DefaultProductUIImageLoader.swift

```diff
@@ -51,31 +54,41 @@ final class DefaultProductUIImageLoader: ProductUIImageLoader {
         }
     }
 
-    deinit {
-        activeImageTasks.forEach { $0.cancel() }
-        activeImageTasks.removeAll()
-    }
```

Comment:
❓ Just wondering: don't we need to cancel all background tasks when the loader is deallocated anymore?

Reply:
Good question. 

We avoid taking ownership of the `Task` here as `ProductImageCollectionViewCell` owns the `Task`, and the task will be deallocated/cancelled when the collection view is deallocated. https://github.com/woocommerce/woocommerce-ios/pull/11902/files#diff-435040a6ba1418be8ae0640c6306cb49930877138097038b7c2a06bd361ecd89

Does it sound good to you?